src/main/java/gov/nih/nci/evs/api/service/ElasticLoadService.java

```diff
@@ -66,21 +68,20 @@ int loadConcepts(ElasticLoadConfig config, Terminology terminology, HierarchyUti
 	 * 
 	 * @param app    the application context object
 	 * @param config the config object
+	 * @param cmd
+	 * @return Terminology
 	 * 
 	 *
 	 * @throws Exception the exception
 	 */
-	Terminology getTerminology(ApplicationContext app, ElasticLoadConfig config) throws Exception;
+	Terminology getTerminology(ApplicationContext app, ElasticLoadConfig config, CommandLine cmd) throws Exception;
```

Comment:
For me, it's awkward to pass something like CommandLine down to a handler implementation because these things shouldn't know anything about the calling environment.  What I'd do is have this take a "String inputPath" parameter instead, and then just unravel that at the higher level and pass the value of -d (and for the other handler just ignore it).  

If for some reason this needs to be called not from the main method, a caller would have to construct a mock CommandLine object to pass it in here.

Or even better, the "ElasticLoadConfig" is being passed in, and it has a "setLocation" method that we're not using anymore (it was for non-realtime) - so instead we should repurpose this and it should get set by the main method and then within the directory handler we can just use "config.getLocation" to get the path where to expect the NCI-META files to exist.